install/kubernetes/cilium/values.yaml.tmpl

```diff
@@ -722,6 +722,10 @@ ingressController:
   # from Ingress to the Load Balancer service
   ingressLBAnnotationPrefixes: ['service.beta.kubernetes.io', 'service.kubernetes.io', 'cloud.google.com']
 
+  # -- IngressLBLabels are the label prefixes, which are used to filter labels to propagate
+  # from Ingress to the Load Balancer service
```

Comment:
suggestion: add "Labels matching any of these prefixes will be copied to the corresponding Service."